app/views/boards/_form.html.erb

```diff
@@ -1,10 +1,6 @@
-<%= form_with(model: board, class: 'mt-5 flex flex-col') do |form| %>
-  <div>
-    <%= form.label :name %>
-    <%= form.text_field :name, class: 'shadow' %>
-  </div>
-
-  <%= form.button(:submit, class: 'button mt-5 w-1/5') do %>
+<%= form_with(model: board, class: 'mt-5 flex flex-col w-2/4') do |form| %>
+  <%= render(FieldComponent.new(:name, form: form, placeholder: 'Name', required: true)) %>
+  <%= form.button(:submit, class: 'button mt-5') do %>
```

Comment:
This is just a question, I'm not sure if this actually make sense but can we use `ButtonComponent` here?

Reply:
Totally, nice catch!